test/test_other.py

```diff
@@ -11148,6 +11148,23 @@ def test_setjmp_emulated_casts(self):
     ''')
     self.do_runf('src.c', 'ok\ndone\n', emcc_args=['-sEMULATE_FUNCTION_POINTER_CASTS'])
 
+  def test_setjmp_no_lto(self):
```

Comment:
Perhaps just call this `test_no_lto`.. and add a comment with a link to the bug, since this seems like a very specific regression test.